CHANGES.md

```diff
@@ -1,5 +1,24 @@
 # Changes
 
+## edge-23.4.2
+
+This edge release contains a number of bug fixes.
+
+* CRDs
+  * Fixed Helm uninstall issue for HttpRoute
```

Comment:
This is actually for the CLI (`linkerd uninstall` subcommand), not helm (even though it involves a `helm.sh/chart` label).